src/internal/Subscriber.ts

```diff
@@ -147,6 +147,46 @@ function bind<Fn extends (...args: any[]) => any>(fn: Fn, thisArg: any): Fn {
   return _bind.call(fn, thisArg);
 }
 
+/**
+ * Internal optimization only, DO NOT EXPOSE.
+ * @internal
+ */
+class ConsumerObserver<T> implements Observer<T> {
+  constructor(private partialObserver: Partial<Observer<T>>) {}
+
+  next(value: T): void {
+    if (this.partialObserver.next) {
+      try {
```

Comment:
You could access the `partialObserver` via `this` only once here: `const { partialObserver } = this` or whatever. There's no need to access it twice. Same for the other methods.